system/systemd.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/python
```

Comment:
it needs to match the service_mgr fact, which already is systemd, we might still have systemd/systemctl_properties and other modules, but the main perception of systemd is still as a service manager.